### ISSUE TITLE
Annotations reported on the wrong lines in the PR

### ISSUE THREAD

**Issue by nayaabkhan** [kamilkisiela/graphql-inspector#907]
Test repository PR: https://github.com/nayaabkhan/graphql-inspector-pr-issue/pull/1/files

Observations:

1. It seems that the lines for comments with `#` are not accounted for.
2. The annotation of removal takes the line number from the older version and displays it on the position for the new version.

**Comment by nayaabkhan** [comment]
@kamilkisiela ICYMI. I also would be glad to work on a PR if this is indeed a confirmed bug. What do you think would be a good starting point to look at for a potential fix?

**Comment by kamilkisiela** [comment]
I fixed it in #1372 

https://github.com/kamilkisiela/graphql-inspector/pull/1372/files#annotation_159906321

<img width="1033" alt="Zrzut ekranu 2020-04-7 o 23 01 39" src="https://user-images.githubusercontent.com/8167190/78719160-c7bba080-7923-11ea-87d4-92a776dfb33b.png">

**Comment by kamilkisiela** [comment]
[v1.30.1](https://github.com/kamilkisiela/graphql-inspector/releases/tag/v1.30.1)

**Comment by nayaabkhan** [comment]
@kamilkisiela That is great! Thank you 🙇🏼‍♂️

Just wondering how I can confirm on my [test repo](https://github.com/nayaabkhan/graphql-inspector-pr-issue/pull/1/files)? I should just rerun the checks right?

![Screenshot 2020-04-09 at 11 21 11 AM](https://user-images.githubusercontent.com/234889/78862133-41eb4280-7a54-11ea-98e5-6e80fba893ed.png)

**Comment by nayaabkhan** [comment]
I went ahead and re-run it, the annotation for new type is now in the right place. But the removal annotation is still misplaced:

<img width="993" alt="Screenshot 2020-04-09 at 11 24 37 AM" src="https://user-images.githubusercontent.com/234889/78862472-17e65000-7a55-11ea-8de8-05dc3d9663a7.png">

**Comment by kamilkisiela** [comment]
@nayaabkhan that's because GitHub doesn't allow to put annotations on the previous version of a file. The annotation that is assigned to line 13, it should be there, but on the `schema.graphql` from `master`.

What we need here is to change the target of each "removed" item. For example, when description of type is removed, we need to put an annotation on type. If it's a description of Type.field, it has to land on Type.field but only if field still exists, if it was removed, we need to put it on Type.

Maybe you want to create a Pull Request? I can guide you, answer all the questions. I've got holidays now, so no work till Monday. If not I will take care of it soon.
